exasol_integration_test_docker_environment/docker_db_config/7.0.20/EXAConf

```diff
@@ -0,0 +1,182 @@
+[Global]
```

Comment:
should we add all files in this directory to the generated ones for Github

Reply:
Not sure what you mean. The Github workflow creates a matrix build entry for all directories under 
`exasol_integration_test_docker_environment/docker_db_config`.
Not sure where we generate other input for Github.

Reply:
I mean .gitattributes to hide generated files in Github review

Reply:
done